### PR TITLE
Camera Updates

### PR DESCRIPTION
**Pull request by katiemacaulay**
**Overview**
We added new configurable options for users and updates to the Camera config.

**Changes**
Updated the Camera Configurations
Added descriptions for all new options
Updated Camera as Viewport Camera as well
Changed styling format to match new configurations that take more space.

**Test**
https://monodrive.readthedocs.io/en/camera_updates/monoDrive_home/Camera/

### CODE REVIEW

**Comment by stephendb** [comment]
Focal length is removed from all camera sensors and senor size, please check them all.

File: docs/monoDrive_home/Camera.md [hunk 1]
```diff
@@ -15,157 +15,215 @@ be controlled through each camera's configuration.
...
+    - **include_obb:** If `true`, the actor oriented bounding box information will be included in annotations.
+    - **include_tags:** If `true`, the actor tag information will be included in annotations.
+    - **lane_sampling_distance:** Distance (cm) along the lane we want included in the annotation.
+    - **lane_sampling_frequency:** Interval (cm) along the lane we want to sample annotation points.
+    - **lane_subsampling:** Interval (pixels) between points in image space we want to actually be returned.
+- **camera_matrix:** Computes Field of View (FOV). 
+- **channel_depth:** Sets the total number of channels that will be in the output.
+- **channels:** Used to determine type of image output. For RGB cameras, this should always be `rgba`.
+- **color_filter_array:** [color filter array (bayer)](./#color-filter-arrays-bayer)
+    - **cfa:** The cfa type.
```
Comment (bradleynull) [inline docs/monoDrive_home/Camera.md:118]:
```suggestion
    - **cfa:** The cfa type. Can be any 4 length string consisting of the letters `rgbc`.
```

File: docs/monoDrive_home/Camera.md [hunk 2]
```diff
@@ -15,157 +15,215 @@ be controlled through each camera's configuration.
...
+    - **use_cfa:** If `use_cfa` set to `true`, enables color filter array. 
+- **dynamic_range:** Controls the contrast of the image.
+- **exposure_compensation:** Control of the ISO value in the image via the exposure scale (moves gray point of the image and controls brightness).
+- **fov:** Controls the horizontal angle of view of the camera. The vertical angle will be dynamically calculated based on `stream_dimensions`.
```
Comment (bradleynull) [inline docs/monoDrive_home/Camera.md:122]:
```suggestion
- **fov:** Controls the horizontal angle of view of the camera. The vertical angle will be dynamically calculated based on `stream_dimensions`. If set to `0` then the camera matrix will be used to compute the field-of-view.
```

**Review by bradleynull — CHANGES_REQUESTED** [review]
@katiemacaulay Looking good! Just a few minor updates to the variable descriptions.

**Review by bradleynull — APPROVED** [review]
@katiemacaulay looks good to go. Approved!

**Pull request merged**